### PR TITLE
(fix/bindings) Updated web and node only examples

### PR DESCRIPTION
**Pull request by kwek20**
# Description of change

Fixes https://github.com/iotaledger/streams/issues/83

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Ran examples found in README in `bindings/wasm`

### CODE REVIEW

File: bindings/wasm/examples/node.js [hunk 1]
```diff
@@ -1,6 +1,119 @@
...
+  let auth = new streams.Author(node, seed, options, false);
+
+  console.log("channel address: ", auth.channel_address());
+  console.log("multi branching: ", auth.is_multi_branching());
+
+  let response = await auth.clone().send_announce();
+  let ann_link = response.get_link();
+  console.log("announced at: ", ann_link.to_string());
+
+  let options2 = new streams.SendOptions(1, 9, true, 1);
```
Comment (dominic22) [inline bindings/wasm/examples/node.js:32]: It seems like the constructor of SendOptions has parameters. I've got also an error for this line in typescript.

From the iota_streams_wasm.d.ts:
```
export class SendOptions {
/**
* @param {number} depth
* @param {boolean} local_pow
* @param {number} threads
*/
  constructor(depth: number, local_pow: boolean, threads: number);
```

File: bindings/wasm/examples/node.js [hunk 2]
```diff
@@ -1,6 +1,119 @@
...
+  .then(() => {
+    console.log("Done example");
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+
+async function main() {
+  let node = "https://api.lb-0.testnet.chrysalis2.com/";
+  let options = new streams.SendOptions(1, 9, true, 1);
```
Comment (dominic22) [inline bindings/wasm/examples/node.js:21]: It seems like the constructor of SendOptions has parameters. I've got also an error for this line in typescript.

From the iota_streams_wasm.d.ts:
```
export class SendOptions {
/**
* @param {number} depth
* @param {boolean} local_pow
* @param {number} threads
*/
  constructor(depth: number, local_pow: boolean, threads: number);
```

File: bindings/wasm/examples/node.js [hunk 3]
```diff
@@ -1,6 +1,119 @@
...
+  console.log("channel address: ", auth.channel_address());
+  console.log("multi branching: ", auth.is_multi_branching());
+
+  let response = await auth.clone().send_announce();
+  let ann_link = response.get_link();
+  console.log("announced at: ", ann_link.to_string());
+
+  let options2 = new streams.SendOptions(1, 9, true, 1);
+  let seed2 = make_seed(81);
+  let sub = new streams.Subscriber(node, seed2, options2, false);
```
Comment (dominic22) [inline bindings/wasm/examples/node.js:34]: For the Subscriber the parameters are also different as in the code of line 34. 

From the iota_streams_wasm.d.ts:


```
export class Subscriber {
  ...
/**
* @param {string} node
* @param {string} seed
* @param {SendOptions} options
*/
  constructor(node: string, seed: string, options: SendOptions);
```

Comment (dominic22) [reply bindings/wasm/examples/node.js:34]: I don't know if the last parameter is now part of the options. Could you please adjust this?

**Review by dominic22 — CHANGES_REQUESTED** [review]

File: bindings/wasm/examples/node.js [hunk 4]
```diff
@@ -1,6 +1,119 @@
...
+  console.log("Sending Keyload");
+  response = await auth.clone().send_keyload_for_everyone(ann_link);
+  let keyload_link = response.get_link();
+  console.log("Keyload message at: ", keyload_link.to_string());
+
+  console.log("Subscriber syncing...");
+  await sub.clone().sync_state();
+
+  let public_payload = to_bytes("Public");
+  let masked_payload = to_bytes("Masked");
```
Comment (dominic22) [inline bindings/wasm/examples/node.js:55]: These payloads also have a different type as the method `send_tagged_packet` is expecting.

Comment (kwek20) [reply bindings/wasm/examples/node.js:55]: Just change `to_bytes` first line into `Uint8Array[str.length]` and solved. I suppose thats what you did

File: bindings/wasm/examples/node.js (hunk 3 above)
Comment (kwek20) [reply bindings/wasm/examples/node.js:34]: You can remove the `false` as subscribers get that boolean from the author announce message now

**Review by dominic22 — APPROVED** [review]

**Pull request merged**